sources/packages/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -0,0 +1,74 @@
+import { Injectable } from "@nestjs/common";
+import { RecordDataModelService } from "../../database/data.model.service";
+import { Connection } from "typeorm";
+
+import {
+  Application,
+  StudentAppeal,
+  StudentAppealRequest,
+  StudentAppealStatus,
+  User,
+} from "../../database/entities";
+import { StudentAppealRequestModel } from "./student-appeal.model";
+
+/**
+ * Service layer for Student appeals.
+ */
+@Injectable()
+export class StudentAppealService extends RecordDataModelService<StudentAppeal> {
+  constructor(connection: Connection) {
+    super(connection.getRepository(StudentAppeal));
+  }
+
+  /**
+   * Save student appeals that are requested by the student
+   * @param applicationId
+   * @param userId
```

Comment:
Can we add some commnets? For instance, for user one we did the below in the past.
![image](https://user-images.githubusercontent.com/61259237/159757868-5f97c0c9-eae2-4dd9-89f9-e4aec72a37e6.png)